source/jormungandr/jormungandr/autocomplete/geocodejson.py

```diff
@@ -57,14 +57,18 @@ def __init__(self, **kwargs):
 
     # TODO: To be deleted when bragi is modified
     @delete_attribute_autocomplete()
-    def get(self, request, instance):
+    def get(self, request, instance, shape):
         if not self.external_api:
             raise TechnicalError('global autocomplete not configured')
 
         q = request['q']
         url = '{endpoint}?q={q}'.format(endpoint=self.external_api, q=q)
         try:
-            raw_response = requests.get(url, timeout=self.timeout)
+            if shape:
```

Comment:
Not sure: shouldn't you test `shape is not None` (and default shape to None)
@kinnou02 what's the pythonic way?